pydrive2/settings.py

```diff
@@ -107,6 +108,10 @@ def LoadSettingsFile(filename=SETTINGS_FILE):
             data = load(stream, Loader=Loader)
     except (YAMLError, OSError) as e:
         raise SettingsError(e)
+
+    if not data:
```

Comment:
here and in all other places: PR is too large - can we split and do step by step? First the most essential things (bring the credentials backend code, migrate to it - everything else later). Or was it an essential thing to the problem we are solving in this PR?